ui/src/components/flows/Flows.vue

```diff
@@ -615,6 +625,35 @@
         return row && row.row && row.row.disabled ? "disabled" : "";
     }
 
+    function onLocalSort(column: any) {
```

Comment:
We should not do any local sorting in the `UI`, just send the parameter to `BE` and show what is returned.